content/services/csv/powerdev_open_source_projects.csv

```diff
@@ -75,6 +75,7 @@
 "OpenLibm","PowerPC support for OpenLibm, a high quality, portable, standalone C mathematical library"
 "OpenMandriva","Building an easy to use, yet powerful and state of art, Linux distribution aimed primarily at desktop use"
 "`OpenQuake`_","An Open Source Engine for seismic hazard and risk assessment uses the infrastructure to try porting to POWER architecture and see how a different architecture affects the numerical tests (the code is covered by numerical, functional and qa tests)"
+"OpenShiftCI","Test builds of OpenShift 4.x for Continuous Integration"
```

Comment:
Let's not include the version as that will become stale over time. Also, let's remove `CI` from the name since that's not included in the proper name of the project.
```suggestion
"OpenShift","Test builds of OpenShift for Continuous Integration"
```